repeating_dates.widget_form.inc

```diff
@@ -339,7 +339,7 @@ function _repeating_dates_widget_form($form_state, $instance, $values, $cardinal
     ),
   );
   // Ical: COUNT.
-  $count = $cardinality;
+  $count = isset($widget_settings['default_count']) ? $widget_settings['default_count'] : $cardinality;
```

Comment:
Not sure about that. The widget has a default value, saving the form will set that. The "isset" will always strike, so that check seems redundant.